src/DataViews/FunctionsDataView.cpp

```diff
@@ -248,10 +247,16 @@ void FunctionsDataView::DoFilter() {
 void FunctionsDataView::AddFunctions(
     std::vector<const orbit_client_data::FunctionInfo*> functions) {
   functions_.insert(functions_.end(), functions.begin(), functions.end());
-  indices_.resize(functions_.size());
-  for (size_t i = 0; i < indices_.size(); ++i) {
-    indices_[i] = i;
-  }
+  OnDataChanged();
+}
+
+void FunctionsDataView::RemoveFunctionsOfModule(const std::string& module_path) {
+  functions_.erase(
+      std::remove_if(functions_.begin(), functions_.end(),
+                     [&module_path](const orbit_client_data::FunctionInfo* function_info) {
```

Comment:
Nit: `FunctionInfo` should be enough, we have the using declaration above.

Reply:
Done.